tests/model/test_performance_tag.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from treeherder.perf.models import PerformanceTag
+
+from django.db.utils import IntegrityError
+
+
+def test_performance_tags_cannot_have_duplicate_names(transactional_db):
+    PerformanceTag.objects.create(id = 1, name = 'harness')
```

Comment:
Please add a performance tag to the `test_perf_alert_summary` fixture; this will ensure the other tests capture this schema change